src/evm-canister-client/src/client.rs

```diff
@@ -926,4 +927,15 @@ impl<C: CanisterClient> EvmCanisterClient<C> {
     pub async fn get_blockchain_block_info(&self) -> CanisterClientResult<BlockchainBlockInfo> {
         self.client.query("get_blockchain_block_info", ()).await
     }
+
+    /// Returns the upgrade info paginated by count and offset
+    pub async fn get_upgrade_info_paginated(
+        &self,
+        count: u64,
```

Comment:
Once we had a `Pagination` type due to complaining about argument ordering (getting confused between count, offset orders.

Reply:
Also offset is usize, but on the canister is u64